libs/driver/magento/src/graphql/apollo-cached-operation-converter.ts

```diff
@@ -0,0 +1,12 @@
+import { Operation } from '@apollo/client/core';
+
+/**
+ * A function to be provided via the DAFF_APOLLO_CACHEABLE_OPERATIONS_CONVERTER injection token when Magento
+ * is the backend platform. This function will convert cacheable operations into an equivalent operation
+ * that Magento will recognize as cacheable.
+ */
+export const DaffMagentoApolloCachedOperationConverter = (o: Operation): Operation => {
```

Comment:
Why wouldn't we just expose the ApolloLink? It seems more suited, and intended, for the purpose than a direct mutation of the underlying operation.

Reply:
We could do that, but we would need two ApolloLinks per platform instead of one per platform and one in core. The reason we would need two per platform is to cover the edgecase of using two platforms for one app. If the app-dev uses two platforms, then the ApolloLinks needs to filter on the platform name as well as `DAFF_APOLLO_CACHEABLE_OPERATIONS`. We could try to compress these "two ApolloLinks per platform" into one, but then we would need to always filter on the platform name, which we don't want when only one platform is used.

So I went with one ApolloLink in each platform (not done in this PR) to cover the edge case, plus one ApolloLink in core instead of two ApolloLinks in each `@daffodil/driver/${platform}`. I could see it either way though, honestly. If you prefer it the other way, I can change it. The naming might get a little tricky, but it can be cleared up with a good doc